package.json

```
@@ -38,20 +39,28 @@
     "clsx": "^2.1.1",
+    "date-fns": "^4.1.0",
     "firebase": "^11.10.0",
     "firebase-admin": "^13.4.0",
+    "framer-motion": "^12.23.0",
     "lucide-react": "^0.525.0",
     "next": "15.3.4",
     "ogl": "^1.0.11",
     "postcss": "^8.5.6",
     "react": "^19.0.0",
+    "react-chartjs-2": "^5.3.0",
     "react-confetti": "^6.4.0",
```

Biome (1.9.4)

[error] The key react-confetti was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)